openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -76,124 +76,123 @@ <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
     {% endif %}
 
     <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__grade">
-        <div class="ui-staff ui-toggle-visibility {% if expanded_view != 'final-grade' %}is--collapsed {% endif %}">
-            <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+        <button class="ui-staff ui-slidable" aria-expanded="false" id="final_grade_{{ submission.uuid }}">
+            <h2 class="staff-info__title ui-staff__subtitle">
                 <i class="icon fa fa-caret-right" aria-hidden="true"></i>
                 <span>{% trans "Learner's Final Grade" %}</span>
             </h2>
-            <div class="ui-toggle-visibility__content">
-                {% if workflow_status == "done" %}
-                    <p class="staff-info__final__grade__score">
-                    {% with points_earned_string=score.points_earned|stringformat:"s" points_possible_string=score.points_possible|stringformat:"s" %}
-                        {% blocktrans with points_earned='<span class="grade__value__earned">'|safe|add:points_earned_string|add:'</span>'|safe points_possible='<span class="grade__value__potential">'|safe|add:points_possible_string|add:'</span>'|safe %}
-                            Final grade: {{ points_earned }} out of {{ points_possible }}
-                        {% endblocktrans %}
-                    {% endwith %}
-                    </p>
-                    <table class="staff-info__status__table staff-info__final__grade__table" summary="{% trans "Final Grade Details" %}">
-                        <thead>
+        </button>
+        <div class="ui-slidable__content" aria-labelledby="final_grade_{{ submission.uuid }}">
+            {% if workflow_status == "done" %}
+                <div class="staff-info__final__grade__score">
+                {% with points_earned_string=score.points_earned|stringformat:"s" points_possible_string=score.points_possible|stringformat:"s" %}
+                    {% blocktrans with points_earned='<span class="grade__value__earned">'|safe|add:points_earned_string|add:'</span>'|safe points_possible='<span class="grade__value__potential">'|safe|add:points_possible_string|add:'</span>'|safe %}
+                        Final grade: {{ points_earned }} out of {{ points_possible }}
+                    {% endblocktrans %}
+                {% endwith %}
+                </div>
+                <table class="staff-info__status__table staff-info__final__grade__table">
+                    <caption class="sr">{% trans "Final Grade Details" %}</caption>
```

Comment:
Note to reviewers: caption is needed for screen readers. And summary wasn't being used properly before (you don't want to have both repeating the same information).